src/Native/Runtime/module.cpp

```diff
@@ -1270,3 +1270,49 @@ UInt32 GenericInstanceDesc::DacSize(TADDR addr)
     return GenericInstanceDesc::GetSize(dummyDesc.GetFlags(), arity);
 }
 #endif // DACCESS_COMPILE
+
+/* static */
+ModuleInfo * ModuleInfo::Create(void * pHeaderStart, void * pHeaderEnd)
```

Comment:
XXXInfo in our system is typically a blob read-only data - GCInfo, UnwindInfo, EHInfo, ... .

I think a better name for this would be ModuleManager. Also, it may be nice to put it into separate .h/.cpp file.